libnixt/include/nixt/Value.h

```diff
@@ -70,4 +70,9 @@ selectStringViews(nix::EvalState &State, nix::Value &V,
   return selectSymbols(State, V, toSymbols(State.symbols, AttrPath));
 }
 
+/// \brief Get nix's `builtins` constant
+inline nix::Value &getBuiltins(const nix::EvalState &State) {
+  return *State.baseEnv.values[0];
+}
+
```

Comment:
Nix should provide this, so I've opened https://github.com/NixOS/nix/pull/11914
Feel free to open PRs to fix tech debt like that